morph_tool/nrnhines.py

```diff
@@ -278,7 +278,25 @@ class NestedPool(multiprocessing.pool.Pool):  # pylint: disable=abstract-method
 
     class Process(multiprocessing.Process):
         """Class that represents a non-daemon process"""
-        daemon = False
+
+        def __init__(self, *args, **kwargs):
+            """Ensures group=None to avoid exception from multiprocessing.
+
+            Note:
+                The exception raised by multiprocessing is:
+                AssertionError: group argument must be None for now
+            """
+            if "group" in kwargs:
+                kwargs.pop("group", None)
+            super().__init__(None, *args[1:], **kwargs)
```

Comment:
I rather meant this
```suggestion
            if "group" in kwargs:
                kwargs.pop("group", None)
            else:
                args.pop(0)
            super().__init__(None, *args, **kwargs)
```
This way it is clear that `group` is the first argument without looking into documentation.

Reply:
But now I'm thinking. If this is a repeated pattern then it's better keep it as it was. However it's a bad pattern IMO.

Reply:
The `args` are often empty here, so the `args.pop(0)` will fail.
And having another `if` to check the length of args would be very heavy, for both readability and perf.

Reply:
if `group` not in `kwargs` then it is in args, no? Anyway, lets keep the old code.

Reply:
I mean this one
```
kwargs.pop("group", None)
super().__init__(None, *args[1:], **kwargs)
```

Reply:
Ok, done :)